geo-types/src/line.rs

```diff
@@ -1,46 +1,76 @@
-use crate::{CoordNum, Coordinate, Point};
+use crate::{CoordNum, Coordinate, NoValue, Point};
+#[cfg(doc)]
+use crate::{Coordinate3D, Coordinate3DM, CoordinateM};
 #[cfg(any(feature = "approx", test))]
 use approx::{AbsDiffEq, RelativeEq};
 
-/// A line segment made up of exactly two
-/// [`Coordinate`s](struct.Coordinate.html).
+/// A generic line segment made up of exactly two [Coordinate] values.
```

Comment:
It's not clear what "generic" means in this context. It was generic before (<T>), I'd say just omit the word.

(and please restore the backticks around [`Coordinate`])